client/verta/verta/_cli/deployment/update.py

```diff
@@ -70,9 +72,14 @@ def update_endpoint(path, run_id, model_version_id, strategy, canary_rule, canar
         for rule in canary_rule:
             strategy_obj.add_rule(_UpdateRule._from_dict(json.loads(rule)))
 
+    if resources:
+        resources_obj = _Resource._from_dict(json.loads(resources))
+    else:
+        resources_obj = None
```

Comment:
Aiya, that's really awkward how `_Resource._from_dict()` returns a _list_ of `_Resource`s, but it's kind of necessary because of the implementation.

Could you add a `# NOTE: ` to that method explaining that it returns a list of itself? I had to take a couple glances to realize it

Reply:
Done. I also renamed resources_obj to resources_list for more clarity.